### DISCUSSION TITLE
Just generate, skip pushing

### DISCUSSION THREAD

**Discussion by Hans5958** [lowlighter/metrics#991] [Q&A] [answered]
Is it possible to only generate the images and not auto-push it?

**Comment by spenserblack** [comment, marked as answer]
The `output_action` input should help with this. See [Configuring Output Option](https://github.com/lowlighter/metrics/tree/master/source/plugins/core#-configuring-output-action).

https://github.com/lowlighter/metrics/blob/70202c4ebc49ebe77f88e879d126f304261d6695/source/plugins/core/metadata.yml#L97-L106

**Reply by Hans5958** [reply]
Ah, I see. Thanks, man.
